packages/devtools_app/lib/src/theme.dart

```diff
@@ -286,6 +286,15 @@ extension DevToolsColorScheme on ColorScheme {
         decoration: TextDecoration.none,
       );
 
+  // Last allocation timestamp displayed.
+  TextStyle get trackTimestampTextStyle => TextStyle(
```

Comment:
since this is in the theme, can you give it a more general name? like subtleItalic or something similar?

Reply:
also is it necessary to set the font size?

Reply:
Sure will rename, this size looked best.

Reply:
italicTextStyle